### PR TITLE
RHINENG-10940: block bootc systems from templates

### PR DESCRIPTION
**Pull request by psegedy**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by psegedy** [comment]
still fails to deploy content-sources
```
clowdapp/content-sources-backend not ready, status conditions
```

File: manager/controllers/template_systems_update.go [hunk 1]
```diff
@@ -75,17 +76,22 @@ func assignTemplateSystems(c *gin.Context, db *gorm.DB, accountID int, template
 	tx := db.Begin()
 	defer tx.Rollback()
 
-	missingIDs, satelliteManagedIDs, err := checkInventoryIDs(db, accountID, inventoryIDs, groups)
+	missingIDs, satelliteManagedIDs, bootcIDs, err := checkInventoryIDs(db, accountID, inventoryIDs, groups)
```
Comment (MichaelMraka) [inline manager/controllers/template_systems_update.go:79]: Would it make sense to return only single list of ids and their error reasons?

**Review by MichaelMraka — APPROVED** [review]

**Pull request merged**